sky/packages/sky/lib/rendering/object.dart

```diff
@@ -215,6 +266,11 @@ class PaintingContext {
       ..isAntiAlias = false;
   }
 
+  /// Paint a child with a color filter
```

Comment:
Since the arguments don't include a color filter, document how the color filter is built from the arguments.
Also document bounds.